chain/client/src/stateless_validation/state_witness_producer.rs

```diff
@@ -45,14 +45,16 @@ impl Client {
             .ordered_chunk_validators();
 
         let my_signer = self.validator_signer.as_ref().ok_or(Error::NotAValidator)?.clone();
+        let witness_size;
         let witness = {
             let witness_inner = self.create_state_witness_inner(
                 prev_block_header,
                 prev_chunk_header,
                 chunk,
                 transactions_storage_proof,
             )?;
-            let (signature, witness_size) = my_signer.sign_chunk_state_witness(&witness_inner);
+            let (signature, _witness_size) = my_signer.sign_chunk_state_witness(&witness_inner);
```

Comment:
generally variables with underscore at the beginning should not be used by rust conventions